portal/assets/js/global.js

```diff
@@ -68,4 +68,21 @@ document.addEventListener('DOMContentLoaded', function() {
     addMaildrop.parentNode.insertBefore(inputDropClone, addMaildrop);
   });
 
+  // FIXME - I don't understand what this do ...
+  // This looks kinda hackish :|
+  if(window.location != window.parent.location) {
```

Comment:
This condition is true, when the page is inside the iframe (page url != html url)